src/index.js

```diff
@@ -2228,6 +2351,7 @@ const createScatterplot = (initialProperties = {}) => {
     regl.poll();
   };
 
+  /** @type {<Key extends keyof Properties>(property: Key) => Properties[Key] } */
```

Comment:
Similar here, TS will infer the return type based on the property name:

- List all properties, can tab complete:

<img width="648" alt="image" src="https://user-images.githubusercontent.com/24403730/123801061-e5be4980-d8b7-11eb-8cac-ec3cf17aea20.png">


- Auto infer return type:

<img width="687" alt="image" src="https://user-images.githubusercontent.com/24403730/123801836-a5130000-d8b8-11eb-85a4-095ff2ce3918.png">